ui/README.md

```diff
@@ -0,0 +1,30 @@
+# React + TypeScript + Vite
+
+This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
```

Comment:
What is HMR? Mabe just link HMR (and also ESLint) to their homepages for completeness?

Reply:
I guess this is an autogenerated README that needs to be replaced/amended later on?